OpenRA.Game/Map/MapPreview.cs

```diff
@@ -253,6 +253,7 @@ public void Install()
 				{
 					Console.WriteLine(e.Message);
 					Status = MapStatus.DownloadError;
+					download.Cancel();
```

Comment:
An exception can potentially occur before line 250, which means that `download` can be `null` here. Please add a null check for extra safety.